pkg/config/setup/config_nix.go

```diff
@@ -14,10 +14,13 @@ var (
 	// InstallPath is the default install path for the agent
 	// It might be overridden at build time
 	InstallPath = "/opt/datadog-agent"
+
+	// defaultRunPath is the default path for the agent's runtime files
+	// It might be overridden at build time
+	defaultRunPath = "/opt/datadog-agent/run"
```

Comment:
Golang does not support overriding fields that are resolved at runtime so the override wasn't working